common/changes/office-ui-fabric-react/shield-PanelBreakpoin_2019-02-14-06-37.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Panel: fixes several screen breakpoint issues and brings more clarity to docs and logic of applying the breakpoints.",
+      "type": "patch"
```

Comment:
I know you're "fixing issues" here, but this change looks big enough I think would be minor since you are doing some breakpoint changes.

Reply:
Agree... I was kinda on the edge here. At the same time, not sure if fixing style bugs should go as minor. Will change it to minor as I think a breakpoint change could have some side effects for  some consumers who relied on them))

Reply:
Changed the type of the change to minor.